### ISSUE TITLE
"undefined" file appears when editing documents from another tree

### ISSUE THREAD

**Issue by agm1984** [icetee/remote-ftp#623]
![remote-ftp](https://cloud.githubusercontent.com/assets/17655219/21578980/c19f2aa8-cf4b-11e6-9cee-3e7a7ee6d1f8.png)

Steps to reproduce:
1) open a folder in Atom
2) install remote-ftp or otherwise toggle it on
3) edit a file in the local tree
4) save it
5) remote-ftp always creates a file called 'undefined' which is inaccessible through Atom
6) it also uploads the file to my server
7) 
![remote-ftp2](https://cloud.githubusercontent.com/assets/17655219/21579010/dc84fe78-cf4c-11e6-815c-3751d5abc60a.png)
8) i was working around it before but this morning, i disconnected remote-ftp and edited a file in my local tree to add a comment and then when I reconnected remote-ftp, it auto uploaded the file again.

Maybe it is a feature and I'm not using remote-ftp properly, but it seems like a bug because I only work out of the remote directory and local files from the other tree are not related.

Thanks,
Adam

**Comment by jpxd** [comment]
Yeah, Remote-FTP has problems with projects with more than one project folder....
